src/components/WheelSelect/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useRef } from 'react';
+
+import Slider from 'react-slick';
+
+import slickSettings from 'components/WheelSelect/slickSettings';
+
+const WheelSelect = ({ data, multipleSelect }) => {
+  const sliderRef = useRef(null);
+  const elementTextClass = multipleSelect
+    ? 'wheel-select__text wheel-select__text--multiple-select'
+    : 'wheel-select__text';
+
+  useEffect(() => {
+    const selectedElement = document.getElementById(
+      'wheel-select__container--0'
+    );
+    selectedElement.style.opacity = 1;
+
+    window.addEventListener('wheel', (e) => {
```

Comment:
- We are initializing this event listener here so that we can change the slick slide when the user scrolls. The slick doesn't have any internal scroll listener so we are creating a listener here.
- It doesn't have to be in the `useEffect`. Moved from `useEffect`
- I am not sure about guidelines, but we put it in the component so that when it is mounted the event listener is registered. 
@Lahphim